mythril/laser/plugin/plugins/plugin_annotations.py

```diff
@@ -67,3 +86,30 @@ def __copy__(self):
         result = WSDependencyAnnotation()
         result.annotations_stack = copy(self.annotations_stack)
         return result
+
+    def check_merge_annotation(self, annotation: "WSDependencyAnnotation") -> bool:
+        if len(self.annotations_stack) != len(annotation.annotations_stack):
+            return False
+        for a1, a2 in zip(self.annotations_stack, annotation.annotations_stack):
+            if a1 == a2:
+                continue
+            if type(a1) != type(a2):
+                return False
+            try:
```

Comment:
```suggestion
            if isinstance(a1, MergeableAnnotation) and isinstance(a2, MergeableAnnotation)
```
It is probably cleaner to perform the check like this vs waiting on an exception 😅